Sources/TuistSupport/XcodeBuild/XcodeBuildController.swift

```diff
@@ -1,8 +1,69 @@
 import Foundation
 import RxSwift
 import TSCBasic
-import TuistCore
-import TuistSupport
+
+public enum XcodeBuildDestination: Equatable {
+    case device(String)
+    case mac
+}
+
+public protocol XcodeBuildControlling {
```

Comment:
This isn't related to this PR, but I'm noticing that this isn't updated to include dSYMs and bcsymbolmaps in xcframeworks

Reply:
Was it solved?